Common/Settings.ini

```diff
@@ -234,6 +234,9 @@ FixDrawingTextLine = 1
 ; Consistency fix for the game's FMV framerate. 
 FixFMVSpeed = 1 
 
+; Enables sixty FPS mode 
```

Comment:
Okay to request a description change?
`Runs the game at 60 FPS, resulting in an overall smoother gameplay experience, while fixing many high framerate issues that would otherwise occur.`